src/components/species-map.js

```diff
@@ -1,7 +1,10 @@
 export default {
```

Comment:
I don't believe this file is needed anymore.

Reply:
Sure, it is now removed.